be/src/exec/vectorized/aggregate/agg_hash_map.h

```diff
@@ -162,6 +183,42 @@ struct AggHashMapWithOneNullableNumberKey {
         }
     }
 
+    template <typename Func>
+    void compute_agg_states_with_limit(size_t chunk_size, const Columns& key_columns, MemPool* pool,
```

Comment:
looks like this piece of code can be simplified, because I guess difference is `_handle_data_key_column`